llm-servers/deepsparse/config.yaml

```diff
@@ -0,0 +1,3 @@
+endpoints:
```

Comment:
this file is not being used at the moment, right? as the deployment is using the args instead of the config option

Reply:
Good catch, forgot to remove it, thanks!